tests/components/orderentry/OrdersTable.test.jsx

```diff
@@ -199,22 +211,31 @@ describe('getUUID() method', () => {
   });
 });
 
-describe('setDiscontinuedDrugOrder method', () => {
-  it('should call setDiscontinuedDrugOrder', async (done) => {
+describe('setDiscontinuedOrder method for drug orders', () => {
+  it('should call setDiscontinuedOrder for drug orders', async (done) => {
+    const renderedComponent = getComponent().instance();
+    sinon.spy(renderedComponent, 'setDiscontinuedOrder');
+    renderedComponent.setDiscontinuedOrder(mockDrugOrder);
+    expect(renderedComponent.setDiscontinuedOrder.calledOnce).toEqual(true);
+    done();
+  });
+});
+
+describe('setDiscontinuedOrder method for lab orders', () => {
+  it('should call setDiscontinuedOrder for lab orders', async (done) => {
     const renderedComponent = getComponent().instance();
-    sinon.spy(renderedComponent, 'setDiscontinuedDrugOrder');
-    renderedComponent.setDiscontinuedDrugOrder(mockOrder);
-    expect(renderedComponent.setDiscontinuedDrugOrder.calledOnce).toEqual(true);
+    renderedComponent.setDiscontinuedOrder(mockLabOrder);
+    expect(renderedComponent.setDiscontinuedOrder.calledTwice).toEqual(true);
```

Comment:
refer to above comment

Reply:
I think a new story would have to be created to fix this as it is out of scope of this task as I did not write those cases.